src/asmcnc/skavaUI/screen_stop_or_resume_decision.py

```diff
@@ -166,7 +168,25 @@ def popup_help(self):
                 self.l.get_bold("Or visit <URL>").replace('<URL>', 'www.yetitool.com/support > Knowledge Base')
             )
 
-            popup_info.PopupQRInfo(self.sm, self.l, 500, info, self.qr_source)
+            qr_popup = BasicPopup(
+                sm=self.sm, m=self.m, l=self.l,
+                title='',
+                main_string=info,
+                popup_type=PopupType.QR,
+                popup_image=self.qr_source,
+                popup_image_size_hint=(1, 1),
```

Comment:
<img width="598" alt="Screenshot 2023-12-09 at 11 15 08" src="https://github.com/YetiTool/easycut-smartbench/assets/47244859/46d45c60-e61e-422e-aaed-876a1fd5fae0">

In original popupQRcodeinfo class the image size_hint_y = 1.5 - is this correct?

Reply:
when I used 1.5, it moved the image to the right side of the popup, didn't look right